packages/apps/composer-app/src/main.tsx

```diff
@@ -261,13 +261,13 @@ const main = async () => {
       [SettingsMeta.id]: Plugin.lazy(() => import('@braneframe/plugin-settings')),
       [SketchMeta.id]: Plugin.lazy(() => import('@braneframe/plugin-sketch')),
       [SpaceMeta.id]: Plugin.lazy(() => import('@braneframe/plugin-space'), {
-        onFirstRun: async ({ personalSpaceFolder, dispatch }) => {
+        onFirstRun: async ({ defaultSpaceRoot, dispatch }) => {
```

Comment:
fyi on collections branch I've removed this arg altogether because it's easily accessible via space properties if its needed